components/navbars.py

```diff
@@ -34,7 +36,7 @@ def is_logged_out(self):
 
 
 class AbstractLegacyEmberNavbar(Navbar):
-    user_dropdown = Locator(By.CSS_SELECTOR, '#secondary-navigation > ul > li:nth-last-of-type(1) > a', settings.LONG_TIMEOUT)
+    user_dropdown = Locator(By.CSS_SELECTOR, 'ul.navbar-nav > li:nth-child(6) > a')
```

Comment:
```suggestion
     user_dropdown = Locator(By.CSS_SELECTOR, 'ul.navbar-nav > li:nth-child(6) > a' , settings.LONG_TIMEOUT)
```
You might find this handy if this dropdown is loading slower than expected